src/hotel-management-app/src/app/restaurant/order-management/order-management.component.ts

```diff
@@ -0,0 +1,41 @@
+// src/app/components/order-management/order-management.component.ts
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Location } from '@angular/common';
+import { CartService } from '../../services/cart.service';
+import { Order } from '../../models/orders.model';
+
+@Component({
+  selector: 'app-order-management',
+  templateUrl: './order-management.component.html',
+  styleUrls: ['./order-management.component.css']
+})
+export class OrderManagementComponent implements OnInit {
+  restaurantId!: number;
+  orders: Order[] = [];
+
+  constructor(
+    private route: ActivatedRoute,
+    private cartService: CartService,
+    private location: Location
+  ) {}
+
+  ngOnInit(): void {
+    this.restaurantId = +this.route.snapshot.paramMap.get('restaurantId')!;
+    this.orders = this.cartService.getOrdersByRestaurant(this.restaurantId);
+  }
+
+  goBack(): void {
+    this.location.back();
+  }
+
+  updateOrderStatus(orderId: number, status: 'Pending' | 'Accepted' | 'Rejected' | 'Completed'): void {
```

Comment:
- [ ] can this type string be created as a type and used instead bof using it directly

Reply:
resolved